requirements.txt

```diff
@@ -1,2 +1,2 @@
 ops
-git+https://github.com/justinmclark/resource-oci-image/@301bbc80927f989bbfaac5e951b6313bcd9ac105#egg=oci-image
\ No newline at end of file
+git+https://github.com/juju-solutions/resource-oci-image/@25b480913c250b07be2821c05fd47e324ff75ae8#egg=oci-image
```

Comment:
Is it okay to use a particular commit?

Reply:
I've wondered the same thing, but this pattern is used in other charms. e.g. https://github.com/charmed-kubernetes/metallb-operator/tree/master/charms/metallb-controller